alpine.docker

```diff
@@ -25,6 +25,9 @@ RUN apk add --update 'gnupg<2.1.17-r0' && \
 # Customized from submodules
 COPY ./alpine/nginx.conf /etc/nginx/nginx.conf
 
+# Enables Pi-hole Teleporter (extract your lists)
+COPY ./alpine/php.ini /etc/php5/php.ini
```

Comment:
Hmm.. Maybe.. When I ran the latest alpine docker build, the gui complained that the zip.so extension wasn't loaded..  So yea, you have it done the right way.

Maybe need to make sure the php.ini has
include=/etc/php5/conf.d
?

I'll rebuild it again from master and see what happens.